deid/events.sql

```diff
@@ -593,4 +593,147 @@ select /*+ PARALLEL(bcarrier_demo_codes,12) */
   'EXTRACT_DT' COL_DT,
   EXTRACT_DT DT
 from bcarrier_demo_codes;
-commit;
\ No newline at end of file
+commit;
+
+
+insert /*+ APPEND */ into date_events
+select /*+ PARALLEL(outpatient_demo_codes,12) */
+'outpatient_demo_codes' table_name,
+  bene_id,
+  null msis_id,
+  null state_cd,
+  'EXTRACT_DT' COL_DT,
+  EXTRACT_DT DT
+from outpatient_demo_codes;
+commit;
+
+insert /*+ APPEND */ into date_events
+with dates as (
+select /*+ PARALLEL(maxdata_ip,12) */
+'maxdata_ip' table_name,
+  bene_id,
+  msis_id,
+  state_cd,
+  EL_DOB,
+  PYMT_DT,
+  ADMSN_DT,
+  SRVC_BGN_DT,
+  SRVC_END_DT,
+  PRNCPL_PRCDR_DT,
+  EXTRACT_DT
+from maxdata_ip

```

Comment:
Again, why all the new code for medicaid tables that have been there from the start?